Vagrantfile

```diff
@@ -11,16 +11,18 @@ SCRIPT
 Vagrant.configure("2") do |config|
   config.vm.define :windows do |windows|
     windows.vm.box = "mwrock/WindowsNano"
-    windows.vm.network :forwarded_port, guest: 22, host: 2222, id: "ssh"
     windows.vm.guest = :windows
+    windows.vm.communicator = "winrm"
+    windows.vm.network :forwarded_port, guest: 22, host: 2222, id: "ssh", disabled: true
     windows.vm.provision "file", source: "resources/cert.pfx", destination: 'C:\cert.pfx'
     windows.vm.provision "shell", privileged: true, inline: windows_enable_winrm_ssl
-    windows.vm.communicator = "winrm"
+    windows.vm.provider "virtualbox" do |vb|
+      vb.gui = false
+    end
   end
 
   config.vm.define :linux do |linux|
     linux.vm.box = "bento/centos-6.7"
-    linux.vm.network :forwarded_port, guest: 22, host: 2224, id: "ssh"
```

Comment:
This will break if there are other VMs running. We should pick a less common value than 2222 and set it explicitly.

Reply:
True, but the same is true for the winrm & winrm-ssl, should we pick some less common ports values for those too?

Reply:
I've listed all the required port forwardings explicitly and without the `auto_correct` options, i.e. they are now static. Either the required ports are available or the provisioning fails.